app/src/main/res/layout/activity_signup.xml

```diff
@@ -2,40 +2,26 @@
 <androidx.constraintlayout.widget.ConstraintLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
Shouldn't the signup page also offers the google option ? It seems a bit weird that to sign-up with google, one needs to go to the login page.
I think it should be a quick fix as the code is already present in login.

Reply:
Yes, I tried to do it, but like you said, it will require some refactoring so I will do it in another PR.